ml/db/init.sql

```diff
@@ -0,0 +1,170 @@
+-- INITALIZATION
+
+DROP DATABASE stratosphere;
+
+CREATE DATABASE stratosphere;
+ALTER DATABASE stratosphere OWNER TO postgres;
+
+\connect stratosphere
+
+ALTER USER postgres PASSWORD 'test1';
+
+-- RANKINGS
+
+-- Ranking relation
+CREATE TABLE ranking (
+	num 				int 		CHECK (num >= 0 AND num <= 36),
```

Comment:
num should be changed to small int. Takes less storage, and there is no possibility that rocket league will have more than 32768.